### ISSUE TITLE
I've had this problem for a while, please fix.

### ISSUE THREAD

**Issue by Mazedotexe** [bypassiwastaken/skiovox-helper#69]
Is there any way to implement Find as Ctrl+F into SH, or bookmarking as Ctrl+D? Saving as Ctrl+S? There are quite a few keyboard shortcuts that I very commonly use, and, since I've never made an extension, I'm not even sure if you can implement these, but it would be very nice. Similarly, I'm not sure if it's possible, but it would be really nice to be able to right click as right now you can only right click on text. It's been this way for a while, but I didn't want to be a nuisance as I feel I've commonly been lol. Thank you in advance!

**Comment by 3kh0** [comment]
I am pretty sure right click is not possible due to the fact we are using kiosk apps.

**Comment by Mazedotexe** [comment]
> I am pretty sure right click is not possible due to the fact we are using kiosk apps.

Yeah, that makes sense. Like I said, I have no idea what you can even do with extensions.

**Comment by pikapower9080** [comment]
By the way, you can still use the function of Control+F by clicking the top right menu and clicking Find.

**Comment by Mazedotexe** [comment]
> By the way, you can still use the function of Control+F by clicking the top right menu and clicking Find.

i know, if i didnt know this then i wouldve made this issue a lot sooner. its just a lot more time consuming to do it manually with the menu. same for save as (ctrl+s).

**Comment by bypassiwastaken** [comment]
ctrl-s and ctrl-d are the only possible ones i see there. if anyone submits a pr ill consider

**Comment by Mazedotexe** [comment]
> ctrl-s and ctrl-d are the only possible ones i see there. if anyone submits a pr ill consider

i would but i mainly only know javascript lol. also, just curious, why is creating bookmarks not possible? do extensions just not have access to those?

Edit: I mean ctrl+f for find. I was wondering why you couldn't do Ctrl-f becaue he said only ctrl-s and ctrl-d were working. My brain was just not working

**Comment by DanTCA1** [comment]
> > ctrl-s and ctrl-d are the only possible ones i see there. if anyone submits a pr ill consider
> 
> i would but i mainly only know javascript lol. also, just curious, why is creating bookmarks not possible? do extensions just not have access to those?

You can, just the bookmark bar is default off, click the three dots in the top right to turn it on

**Comment by Mazedotexe** [comment]
> > > ctrl-s and ctrl-d are the only possible ones i see there. if anyone submits a pr ill consider
> > 
> > 
> > i would but i mainly only know javascript lol. also, just curious, why is creating bookmarks not possible? do extensions just not have access to those?
> 
> You can, just the bookmark bar is default off, click the three dots in the top right to turn it on

Oh ik, I actually did that before I even installed Skiovox Helper lol

**Comment by hermanboxcar5** [comment]
yes ctrl f PLEASE

**Comment by blankuserrr** [comment]
> yes ctrl f PLEASE

can't and won't ever work
